src/main/webapp/i18n/en/editor.json

```
@@ -34,6 +34,7 @@
             "buildOutput": "&nbsp; Build Output",
             "building": "Building and testing...",
             "buildFailed": "Build Failed",
+            "buildSuccessful": "Build Successful, no Tests Executed",
```

We should probably be consistent with capitalization here, so either `No` or perhaps
```suggestion
            "buildSuccessful": "Build Successful, no tests executed",
```
?